plugins/auth-core/src/index.ts

```diff
@@ -156,7 +157,24 @@ class AuthCorePlugin implements AmplicationPlugin {
     context: DsgContext,
     eventParams: CreateServerPackageJsonParams
   ) {
-    context.utils.skipDefaultBehavior = true;
+    const myValues = {
+      dependencies: {
+        "@nestjs/jwt": "8.0.0",
+        "@nestjs/passport": "8.2.2",
+        "passport": "0.6.0",
+        "passport-http": "0.3.0",
+        "passport-jwt": "4.0.0",
```

Comment:
let's update to the latest `passport-jwt` otherwise we bring back the sec vulnerability CVE-2022-3517